src/data/hicdex.js

```diff
@@ -207,3 +242,37 @@ export async function fetchGraphQL(operationsDoc, operationName, variables) {
 
   return await result.json()
 }
+
+export async function getObjktsByShare(addresses, min_shares) {
+  const { errors, data } = await fetchGraphQL(
+    getCollabTokensForAddressesByShare,
+    'GetCollabTokens',
+    { addresses }
+  )
+
+  if (errors) {
+    console.error(errors)
```

Comment:
maybe throw or return here, because the code that follows expects a `data` object and will fail if it's undefined.

Reply:
Yes tbh I copied the other methods but I usually try/catch those. We could also use the graph-ql package, it's very nice, I'm actually working on a little library wrapper that use most of the request we make here, I will propose it once I'm done :)

Reply:
Hmmm how are we handling them then? I never seem to be able to hit an error (providing a wrong address for instance).
I think this is because how it's handled in fetchGraphQL
I think it's out of scope for this issue